_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
+    * [ ] [Error & performance alerts](https://pages.18f.gov/before-you-ship/infrastructure/monitoring/#errors--performance-problems)
+- [ ] Perform security scans, and put the results (or a link to them) in the ATO folder.
     * [ ] Set up [static analysis](https://pages.18f.gov/before-you-ship/security/static-analysis/) service
         * [ ] Add service badges to the README
+        * [ ] Put a point-in-time PDF of the scan results in the ATO folder
     * [ ] [Perform dynamic vulnerability scanning](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
         * [ ] Resolve any visible security issues, re-running the scan as needed
         * [ ] Add the issue-free scan report or [documentation about false positives](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/#caveats) to the ATO folder.
+- [ ] If this is a new system, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to the site
+- [ ] Ensure the staging environment has sufficient capacity to withstand the testing
+    * The testing tools create very heavy usage and traffic.
+
+##### Content
+
+- [ ] Read the LATO guide<!-- unless not doing a LATO -->.
+    * Search [this page](https://insite.gsa.gov/portal/content/627230) for "Lightweight Security Authorization Process".
+- [ ] Create a [system diagram](https://pages.18f.gov/before-you-ship/ato/ssp/#systemnetwork-diagrams)
+- [ ] Fill out the Rules of Engagement (RoE)
+    * Use staging URLs, rather than production ones.
+- [ ] Fill out the Project Plan
+- [ ] Add an [`.about.yml`](https://github.com/18F/about_yml) for the main repository
 - [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README
-- [ ] If this will be a new ATO, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to any currently-running sites
-- [ ] Add a [System Security Plan yml](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
-- [ ] Add a System Security Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
+- [ ] Add a [System Security Plan YAML file](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
 - [ ] [Set up Compliance Masonry documentation](https://github.com/18F/cg-compliance#starting-ato-documentation-for-cloudgov-applications)
-- [ ] [Implement the controls](https://pages.18f.gov/before-you-ship/ato/walkthrough/#step-3--implement-the-controls)
-- [ ] Add a Rules of Engagement (RoE) Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
-- [ ] Ensure system environment referenced in the RoE for security testing is ready for testing. No production links should be included in the RoE so as to avoid any testing happening on production (which could lead to downtime).
-- [ ] Add a Project Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
 
-### Authorizing Official
+### Documentation review
```

Comment:
It's not clear that this is a certain point in the process/in time. Maybe a sentence between the above and this outlining that now you are entering the sprint process (and a link describing the process)?